### PR TITLE
KEP for CoreDNS to Default DNS

### PR DESCRIPTION
**Pull request by rajansandeep**
This KEP is to defined to make CoreDNS the default cluster DNS in Kubernetes.
It is a follow-up to the previous KEP (https://github.com/kubernetes/community/pull/1956)

/sig network
/assign @thockin 
/assign @bowei

### CODE REVIEW

File: keps/sig-network/0011-20180518-coredns-default-proposal.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+CoreDNS is now well-established in Kubernetes as the DNS service, with CoreDNS starting as an alpha feature from Kubernetes v1.9 to now being GA in v1.11.
+After successfully implementing the road-map defined [here](https://github.com/kubernetes/features/issues/427), CoreDNS is GA in Kubernetes v1.11, which can be installed as an alternate to kube-dns in tools like kubeadm, kops, minikube and kube-up.
+Following the [KEP to graduate CoreDNS to GA](https://github.com/kubernetes/community/pull/1956), the purpose of this proposal is to make CoreDNS as the default DNS for Kubernetes, replacing kube-dns.
+
+## Goals
+* Make CoreDNS the default DNS for Kubernetes for all the remaining install tools (kube-up, kops, minikube).
+* Make CoreDNS available as an image in a Kubernetes repository (To Be Defined) and ensure a workflow/process to update the CoreDNS versions in the future.
+  This goal is carried over from the [previous KEP](https://github.com/kubernetes/community/pull/1956), in case it cannot be completed there.
+
+## Proposal
```
Comment (thockin) [inline keps/sig-network/0011-20180518-coredns-default-proposal.md]: do we also want this to encompass the formal deprecation of kube-dns ?

**Comment by thockin** [comment]
I'm good with this - anyone else?

File: keps/sig-network/0011-20180518-coredns-default-proposal.md (hunk 1 above)
Comment (thockin) [reply keps/sig-network/0011-20180518-coredns-default-proposal.md]: ping

Comment (johnbelamaric) [reply keps/sig-network/0011-20180518-coredns-default-proposal.md]: Yes, I think that makes sense. We don't want to maintain two solutions for any longer than necessary.

File: keps/sig-network/0012-20180518-coredns-default-proposal.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+#### Kops
+
+* CoreDNS will now become the default DNS.
+
+## Graduation Criteria
+
+* Add CoreDNS image in a Kubernetes repository (To Be Defined) and ensure a workflow/process to update the CoreDNS versions in the future.
+* Have a certain number (To Be Defined) of clusters of significant size (To Be Defined) adopting and running CoreDNS as their default DNS.
+
```
Comment (krmayankk) [inline keps/sig-network/0012-20180518-coredns-default-proposal.md]: have migration scripts to provide a migration from kubedns configs to coredns ?

Comment (rajansandeep) [reply keps/sig-network/0012-20180518-coredns-default-proposal.md]: @krmayankk this is already implemented from Kubernetes 1.10 for kubeadm.

File: keps/sig-network/0011-20180518-coredns-default-proposal.md (hunk 1 above)
Comment (thockin) [reply keps/sig-network/0011-20180518-coredns-default-proposal.md]: I guess I was asking it to be in the doc as a goal. :)

**Pull request merged**